filament/src/PerViewUniforms.cpp

```diff
@@ -87,6 +87,16 @@ void PerViewUniforms::prepareCamera(const CameraInfo& camera) noexcept {
     s.clipControl = mClipControl;
 }
 
+void PerViewUniforms::prepareUpscaler(math::float2 scale,
+        DynamicResolutionOptions const& options) noexcept {
+    auto& s = mPerViewUb.edit();
+    if (options.quality >= QualityLevel::HIGH) {
+        s.lodBias = std::log2(std::min(scale.x, scale.y));
```

Comment:
Should we do it any time FSR is active?

Reply:
The FSR doc recommends to do it when FSR is active. Here it's done only at HIGH and  ULTRA quality levels. The idea is that lod bias will impact bandwidth. The quality improvement is very subtle as well.
Also if the scale factor is 1.0 the log2() ends up being 0.0.

Reply:
Right I wasn't worried about scale 1, I was mostly wondering if we should enable it at medium as well. But good point about bandwidth.